.circleci/config.yml

```diff
@@ -92,6 +92,8 @@ commands:
           helm package deployment/lookout-migration/ -d charts/armada/
           helm package deployment/binoculars/ -d charts/armada/
           helm package deployment/jobservice/ -d charts/armada/
+          helm package deployment/lookout-ingester/ -d charts/armada/
```

Comment:
The lookout-ingester stuff was missing from this file so I've added it along with the event-ingester.